.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "xamboni"]
+	path = xamboni
+	url = git@github.com:urbanairship/xamboni.git
```

Comment:
Not sure how I feel about having a submodule. Do we have a different way of pulling it in, maybe with pip?